dhall/src/Dhall/Core.hs

```diff
@@ -72,6 +72,7 @@ module Dhall.Core (
     , escapeText
     , pathCharacter
     , throws
+    , textShow
     ) where
 
 #if MIN_VERSION_base(4,8,0)
```

Comment:
We shouldn't need this anymore: dhall has a constraint `base >=4.8.2.0`.

We might also be able to get rid of the `LANGUAGE CPP` pragma I believe.